test/dx_mgn_pool.js

```diff
@@ -54,6 +62,14 @@ contract("DxMgnPool", (accounts) => {
       const frtToken = dx.contract.methods.frtToken().encodeABI()
       await dxMock.givenMethodReturnAddress(frtToken, mgnTokenMock.address)
 
+      const owlTokenMock = await MockContract.new()
+      const owlToken = dx.contract.methods.owlToken().encodeABI()
+      await dxMock.givenMethodReturnAddress(owlToken, owlTokenMock.address)
+
+      const owl = await TokenOWL.new();
+      const owlTokenApproveFunctionality = owl.contract.methods.approve(dxMock.address, 100).encodeABI()
+      await owlTokenMock.givenMethodReturnBool(owlTokenApproveFunctionality, true)
```

Comment:
Is mocking this really needed given that we don’t check the return value of approve?

Reply:
Yes, calling into an unknown interface causes a revert.